recipes/aioprocessing/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "aioprocessing" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dano/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 1fe91c0dca9d98e1a1280c120a7c29841f2b79fe9789239adb0d93d5200a0b60
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install --use-pep517  . -vvv .
+  number: 0
+
+requirements:
+  host:
+    - python >=3.5
+    - flit-core
+    - pip
+  run:
+    - python >=3.5
+
+test:
+  imports:
+    - aioprocessing
+  source_files:
+    - runtests.py
+    - tests
+  commands:
+    - pip check
+    - python -m coverage run --branch runtests.py -v --failfast
+  requires:
+    - coverage
+    - pip
+    - python <3.10  # version check incorrect for 3.1x.y
```

```suggestion
    - python <3.10  # version check incorrect for 3.1x.y
```
This is a bit weird, can you try this again please? Maybe it got fixed by now.

---

I don't remember the details, but I get this when I run the test:
```
+ python -m coverage run --branch runtests.py -v --failfast
Traceback (most recent call last):
  File "/home/memento/.conda/envs/CONDA_BUILD/conda-bld/aioprocessing_1705534530338/test_tmp/runtests.py", line 42, in <module>
    assert sys.version >= '3.5', 'Please use Python 3.5 or higher.'
AssertionError: Please use Python 3.5 or higher.
Tests failed for aioprocessing-2.0.1-py_0.tar.bz2 - moving package to /home/memento/.conda/envs/CONDA_BUILD/conda-bld/broken
```
The alternative ti to remove the coverage test, I think

---

Ahh ok. We have a higher version now, so it should be fine. Please remove that line then.

---

This goes away if the extra test is removed. It's simpler

---

@BastianZim I think the issue is in how the version is checked by the `runtest.py` file, like maybe a digit issue where the 3.10 is read as 3.1 - I removed the extra testing